client/Combatant/Combatant.ts

```diff
@@ -168,7 +168,12 @@ export class Combatant implements Combatant {
         return modifiers;
     }
 
-    public GetInitiativeRoll = () => this.Encounter.Rules.AbilityCheck(this.InitiativeBonus, this.StatBlock().InitiativeAdvantage ? "advantage" : null);
+    public GetInitiativeRoll(): number {
```

Comment:
Use arrow function syntax:
`public GetInitiativeRoll: () => number = () => {`
The benefit of that syntax is that it prevents the `this` object from being re-bound if you use this method as a free function.

Reply:
I've immediately run into an error saying `Type 'string' is not assignable to type '"None" | "By Name" | "Side Initiative"'.` I see a number of open issues in the TypeScript repo about this. I think I could get around this by defining the list everywhere one of the values could be assigned, but not sure about that. Wondering instead if this would be better implemented as an Enum, and then referencing the enum values elsewhere in the code, possibly as part of the AutoGroupInitiative encapsulation mentioned below. Any ideas on this?